docs/examples/network_policies/README.md

```diff
@@ -0,0 +1,71 @@
+# Network policy examples
+
+## Valid policy configuration
+
+To run this examples apply the `policies.yaml` to your setward namespace.
+
+```
+kubectl -n steward-system apply -f policies.yaml
+```
+
+### Network policy with internet access
+
+This example pipeline run uses a network policy which allows internet access.
+
+```
+kubectl -n steward-system apply -f pipelinerun_network_internet.yaml

```

Comment:
The pipeline run should not be created in the Steward system namespace. Although it technically works, it gives a wrong impression to the reader of this page.

Reply:
See https://github.com/SAP/stewardci-core/pull/188/commits/8a4d0a17c8336f18f8b65d8a44f74fe927b9ed8d